pyramid_zipkin/request_helper.py

```diff
@@ -1,39 +1,16 @@
 # -*- coding: utf-8 -*-
 
-import codecs
-import os
 import re
-import struct
-from collections import namedtuple
 
 import six
 from pyramid.interfaces import IRoutesMapper
+from py_zipkin.zipkin import ZipkinAttrs
```

Comment:
Nit: swap the order of the py_zipkin imports to keep alphabetical ordering. We should add pre-commit hooks.